### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by VisaganR** [mihir-m-gandhi/Adaptive-Traffic-Signal-Timer#7]
_pip install -r requirements.txt_ not run properly -->errror occurs which sub-process exited while backend dependency is run.What can I do sir? Thanks in advance.

**Comment by rahul-0000-code** [comment]
You might be in different directory, it worked fine for me, but there is cython lib related issues

**Comment by donga02** [comment]
yes...same issue for me.. backend dependency probelm...what should i do?

**Comment by mihir-m-gandhi** [comment]
This project is quite old. I'll take a look and update it soon!
